hlabsorg/league-gg
# Lol 39 implement redis cache for riot ids and api requests

## JIRA ticket(s)


https://hlabs.atlassian.net/browse/LOL-39

## Type of Change




## Changes


- add's `ioredis` package
- creates redis client
- creates cache functions
- implements cache on riot request
- implements cache on summoner route


## Screenshots



## Tests


1.  search for summoner
2.  check network tab for speed before cache (`~250-500ms`)
3.  search for another summoner
4.  see response time reduced to `~50-60ms`
